### PR TITLE
Adding post

### PR DESCRIPTION
**Pull request by anudeepannin**

### CODE REVIEW

**Review by lsingh01 — APPROVED** [review]
The changes looks good to me

**Review by 00kailash — APPROVED** [review]

File: postapi/Controllers/AddPostController.cs [hunk 1]
```diff
@@ -9,5 +16,104 @@ namespace postapi.Controllers
...
     {
+
+        private readonly IConfiguration _configuration;
+        public AddPostController(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+        [Route("GetPosts")]
+        [HttpGet]
+        public JsonResult GetPosts()
```
Comment (akshayverma91) [inline postapi/Controllers/AddPostController.cs]: use try and catch block in every action.
use return type IActtionResult to handle multiple return type of action method

**Review by akshayverma91 — APPROVED** [review]

File: postapi/Controllers/AddPostController.cs [hunk 2]
```diff
@@ -9,5 +16,104 @@ namespace postapi.Controllers
...
+                    myCon.Close();
+                }
+            }
+
+            return new JsonResult(table);
+        }
+        [Route("CreatePost")]
+        [HttpPost]
+        public JsonResult CreatePost(Post p1)
+        {
```
Comment (akshayverma91) [inline postapi/Controllers/AddPostController.cs:63]: Do as above comment

**Review by akshayverma91 — CHANGES_REQUESTED** [review]

File: postapi/Controllers/AddPostController.cs (hunk 1 above)
Comment (anudeepannin) [reply postapi/Controllers/AddPostController.cs]: Updated code in the add post

File: postapi/Controllers/AddPostController.cs (hunk 2 above)
Comment (anudeepannin) [reply postapi/Controllers/AddPostController.cs:63]: Updated code in  the add post

Comment (akshayverma91) [reply postapi/Controllers/AddPostController.cs:63]: looks good

**Review by akshayverma91 — APPROVED** [review]

**Pull request merged**